MapboxMobileEvents/MMEEvent.m

```diff
@@ -431,12 +432,17 @@ - (id)copyWithZone:(NSZone *)zone {
 - (instancetype)initWithCoder:(NSCoder *)aDecoder {
     if (self = [self init]) {
         NSInteger encodedVersion = [aDecoder decodeIntegerForKey:MMEEventVersionKey];
-        _dateStorage = [aDecoder decodeObjectOfClass:MMEDate.class forKey:MMEEventDateKey];
-        _attributesStorage = [aDecoder decodeObjectOfClass:NSDictionary.class forKey:MMEEventAttributesKey];
+        
         if (encodedVersion > MMEEventVersion2) {
-            NSLog(@"%@ WARNING encodedVersion %li > MMEEventVersion %li",
-                NSStringFromClass(self.class), (long)encodedVersion, (long)MMEEventVersion1);
+            NSString *errorString = [[NSString alloc] initWithFormat:@"%@ %@ encodedVersion %li > MMEEventVersion %li",
+                                     self.name, NSStringFromClass(self.class), (long)encodedVersion, (long)MMEEventVersion1];
```

Comment:
self.name isn't supposed to ever be nil, but just in case let's use `self.name ?: @"nil"`

Reply:
Good catch